x-pack/metricbeat/module/gcp/timeseries_metadata_collector.go

```diff
@@ -53,14 +57,19 @@ func (s *StackdriverTimeSeriesMetadataCollector) Metadata(ctx context.Context, i
 
 	ecs := mapstr.M{
 		ECSCloud: mapstr.M{
-			ECSCloudAccount: mapstr.M{
+			ECSCloudProject: mapstr.M{
 				ECSCloudAccountID:   accountID,
 				ECSCloudAccountName: accountID,
```

Comment:
@zmoog should we still keep both `cloud.project.id` and `cloud.project.name` with the same value :thinking: ?

Reply:
If we are removing an existing field, that will be a breaking change right?

Reply:
@kaiyan-sheng I was thinking if we can get the actual GCP project number e.g. `550330936418` instead of using the same project name, but that would require another API call maybe.

Reply:
Example of real account details:

![image](https://github.com/user-attachments/assets/c5ca653c-dd80-4ee6-a90d-0be89961609d)

Reply:
ahh good point! Sorry Im not familiar enough but I would say if user uses this field a lot, maybe its worth the extra api call?

Reply:
Since we can get organization and project info using one API call each only when the metricset starts, it's worth doing it.